users-api/routes/signup.js

```diff
@@ -14,17 +14,48 @@ router.get('/testRoute', (req,res) => res.send('Test successful.'));
 // if something was entered for username and password,
 // confirm the signup and display the information
 router.post('/', (req, res) => {
-  if((!req.body.username) || (!req.body.password))
+  if(req.body.username)
   {
-    res.status(403).send('Invalid information.');
-  }
+  if((!req.body.username) || (!req.body.password))
+    {
+   res.status(403).send('Invalid information.');
```

Comment:
We need to use `res.status(403).json({message: "Invalid Information"})` for our UI to parse it correctly